tests/Pest.php

```diff
@@ -1,5 +1,11 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Importing the code previously in `bootstrap.php` into `Pest.php`, since we've moved to that test framework, and it invokes this script before bootstrap. This ensures AUTH0_TEST_DIR is available earlier in test processing and avoids potential pitfalls when setting up beforeAll and afterAll calls.